t/mojo/morbo.t

```diff
@@ -136,6 +136,49 @@ subtest 'Broken symlink' => sub {
   ok !$warned, 'no warnings';
 };
 
+subtest 'Randomized Request IDs' => sub {
+  my @ids;
+  my $src = <<'EOF';
+use Mojolicious::Lite;
+
+app->log->level('fatal');
+
+# Return "request_id"
+get '/request_id' => sub {
+  my $c = shift;
+  $c->render( text => $c->req->request_id );
+};
+
+app->start;
+EOF
+
+  # Fetch initial request_ids
+  $script->spurt($src);
+  sleep 3;
+
+  $tx = $ua->get("http://127.0.0.1:$port/request_id");
+  like $tx->res->body, qr/^[\-_0-9a-z]+$/i, 'okay content';
+  $ids[0] = $tx->res->body;
+
+  $tx = $ua->get("http://127.0.0.1:$port/request_id");
+  like $tx->res->body, qr/^[\-_0-9a-z]+$/i, 'okay content';
+  $ids[1] = $tx->res->body;
+
+  isnt $ids[1], $ids[0], "different (initial) request_ids: \"$ids[0]\" != \"$ids[1]\"";
```

Comment:
Including the ids in the test description adds no value and makes things less readable.

Reply:
If they don't match they will be shown in the output anyway.

Reply:
I think I've been burned by `ok $one eq $two` (rather than `is $one, $two`) too many times...  :-}

That, and by having tests bail out before they actually print the line number, whereas a unique and descriptive test name makes it at least *possible* to grep for what died.  «shrug»